src/Lime.Protocol.UnitTests/LimeUriTests.cs

```diff
@@ -32,6 +33,19 @@ public void Parse_ValidAbsoluteString_ReturnsInstance()
             actual.IsRelative.ShouldBe(false);
         }
 
+        [Test]
+        public void Parse_ValidAbsoluteSpecialCharactersString_ReturnsInstance()
+        {
```

Comment:
Suggestion: another important characteristic of this test is the space on the path, so add 'space' on the test name could be interesting.

Reply:
Nice, will do!